tasks/lula.yaml

```diff
@@ -0,0 +1,45 @@
+variables:
+  - name: OSCAL_FILE
+
+tasks:
+  - name: default
```

Comment:
tasks that get included shouldnt be named default. Realistically this would make sense in the lula repo, but in this repo it should be a meaningful name and then you should have a task in the root that actually tests lula/this task similar to what exists for the other common things.

Reply:
Updated the lula.yaml and tasks.yaml to include the correct names, more meaningful names, and the test.

Modified the code of the test cmd to its a direct copy of the lint task.

Pulled out the first line piece in the cmd that sets the OSCAL_FILE variable from a github action. Made more sense for that to be in the action that calls the task in X repo this gets added too.

Added in an echo and set variable to fill in that variable for the test file so it points to the included oscal.yaml for testing.

Also updated the renovate to reflect the version of lula to match readme.